### PR TITLE
GH-97 Remove litecommands and improve output plugin.yml

### PR DESCRIPTION
**Pull request by vLuckyyy**
WHY REMOVE LITECOMMANDS? BECAUSE IN THIS CODE WE ONLY HAVE ONE COMMAND! (decrased size from 1.5mb to 1.2mb) 🏆 

And i improved output plugin.yml example for lp editor 😊

Fixes: #97

### CODE REVIEW

File: chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java [hunk 1]
```diff
@@ -2,21 +2,26 @@
...
+import java.util.List;
 import java.util.concurrent.TimeUnit;
 
-@Route(name = "chatformatter")
-@Permission("chatformatter.chat.reload")
-class ChatFormatterCommand {
+class ChatFormatterCommand implements CommandExecutor, TabCompleter {
+
+    private static final String RELOAD_MESSAGE = "<b><gradient:#29fbff:#38b3ff>ChatFormatter:</gradient></b> <green>Successfully reloaded configs in %sms!";
+    public static final String RELOAD_PERMISSION = "chatformatter.chat.reload";
```
Comment (Osnixer) [inline chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java:24]: chyba troszku bez sensu proponowałbym:
chatformatter.reload

File: chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterPlugin.java [hunk 2]
```diff
@@ -53,22 +47,17 @@ public ChatFormatterPlugin(Plugin plugin) {
...
 
         // bStats metrics
         new Metrics((JavaPlugin) plugin, 15199);
 
         this.chatHandler = new ChatHandlerImpl(miniMessage, pluginConfig, this.rankProvider, this.placeholderRegistry, this.templateService);
 
-        List.of(
-            new UpdaterController(updaterService, pluginConfig, audienceProvider, miniMessage)
-        ).forEach(listener -> server.getPluginManager().registerEvents(listener, plugin));
+        server.getPluginCommand("chatformatter").setExecutor(new ChatFormatterCommand(configManager, audienceProvider, miniMessage));
```
Comment (Osnixer) [inline chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterPlugin.java:59]: brakuje rejestracji tabcomplater'a

**Review by Osnixer — APPROVED** [review]

File: chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java (hunk 1 above)
Comment (vLuckyyy) [reply chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java:24]: breaking changes

Comment (Osnixer) [reply chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java:24]: > breaking changes

najaak

**Review by imDMK — APPROVED** [review]

**Review by Jakubk15 — APPROVED** [review]

File: chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java [hunk 3]
```diff
@@ -28,19 +33,40 @@ class ChatFormatterCommand {
...
-    void reload(CommandSender commandSender) {
-        Stopwatch stopwatch = Stopwatch.createStarted();
+    @Override
+    public boolean onCommand(CommandSender sender, Command command, String label, String[] args) {
+        if (args.length == 0) {
+            sender.sendMessage(command.getUsage());
+            return true;
+        }
+
+        if (args[0].equalsIgnoreCase("reload") && sender.hasPermission(RELOAD_PERMISSION)) {
```
Comment (Rollczi) [inline chatformatter-core/src/main/java/com/eternalcode/formatter/ChatFormatterCommand.java:43]: if (args.length == 0 || args[0].equalsIgnoreCase("reload")) {
            sender.sendMessage(command.getUsage());
            return true;
        }

File: chatformatter-paper-plugin/build.gradle.kts [hunk 4]
```diff
@@ -12,6 +14,66 @@ bukkit {
...
+                "chatformatter.color.blue",
+                "chatformatter.color.green",
+                "chatformatter.color.aqua",
+                "chatformatter.color.red",
+                "chatformatter.color.light_purple",
+                "chatformatter.color.yellow",
+                "chatformatter.color.white"
+            )
+            default = Default.OP
+        }
```
Comment (Rollczi) [inline chatformatter-paper-plugin/build.gradle.kts:71]: add `chatformatter.*` permission

File: chatformatter-paper-plugin/build.gradle.kts [hunk 5]
```diff
@@ -12,6 +14,66 @@ bukkit {
     name = "ChatFormatter"
     depend = listOf("PlaceholderAPI", "Vault")
     version = "${project.version}"
+
+    commands {
+        register("chatformatter") {
+            description = "Reloads the configs for ChatFormatter"
+            permission = "chatformatter.chat.reload"
```
Comment (Rollczi) [inline chatformatter-paper-plugin/build.gradle.kts:21]: `chatformatter.reload`?

**Review by Rollczi — APPROVED** [review]
Przykra sprawa usuwać virusa.

**Pull request merged**